Gruntfile.js

```diff
@@ -65,7 +65,7 @@ module.exports = function (grunt) {
     uglify: {
       plugins_js: {
         files: [{
-          src: [ 'node_modules/smartmenus/dist/jquery.smartmenus.min.js'],
+          src: [ 'node_modules/smartmenus/dist/jquery.smartmenus.min.js', 'node_modules/lightcase/src/js/lightcase.js'],
```

Comment:
Lets break each of these onto a new line, like the sass globbing stuff on line 82